src/CMakeLists.txt

```diff
@@ -13,13 +13,14 @@ if(NOT CMAKE_BUILD_TYPE AND NOT CMAKE_CONFIGURATION_TYPES)
       STRING "Choose the type of build." FORCE)
   # Set the possible values of build type for cmake-gui
   set_property(CACHE CMAKE_BUILD_TYPE PROPERTY STRINGS
-    "Debug" "Release" "MinSizeRel" "RelWithDebInfo")
+    "Debug" "Release" "MinSizeRel" "RelWithDebInfo" "tsan" "ASAN" "LSAN" "msan" "ubsan")
```

Comment:
This looks odd...

1. Can you standardize the names, e.g., all uppercase?

2. Can you have a separate variable for it? If it is set to None, use CMAKE_BUILD_TYPE, otherwise, overwrite it with the flags for each sanitizer.

Reply:
1. Sure

2. I think that overwriting the Build Type will create confusion for users. Instead, we could add a Build Type for Sanitizers and another variable specifying which will be used. What do you think?

Reply:
I like the idea, it would be nice to set/unset the variable specifying the sanitizer when the buildType is changed, so it disappears from cmake-gui.

Reply:
Done.